services/scan/src/store.ts

```diff
@@ -871,22 +871,28 @@ export class Store {
       const frontImage: InlineBallotImage = { normalized: '' };
       const backImage: InlineBallotImage = { normalized: '' };
       if (
-        frontInterpretation.type === 'InterpretedHmpbPage' ||
-        frontInterpretation.type === 'UninterpretedHmpbPage'
+        asBoolean(
+          process.env['ENABLE_WRITE_IN_ADJUDICATION_EXPORT_BALLOT_IMAGES']
+        )
```

Comment:
What do you think of moving this to a `services/scan/src/config/features.ts` file and wrapping it in a helper function to mirror what we do elsewhere in vxsuite? So that it's better documented:

https://github.com/votingworks/vxsuite/blob/c76e1094caae7fe999186af3d3d3a0cb2cd07220/frontends/election-manager/src/config/features.ts#L45-L60

Once you do this, in tests, you can mock like this, without having to touch `process.env`:

https://github.com/votingworks/vxsuite/blob/c76e1094caae7fe999186af3d3d3a0cb2cd07220/frontends/election-manager/src/components/smartcard_modal/pins.test.ts#L6-L20

https://github.com/votingworks/vxsuite/blob/c76e1094caae7fe999186af3d3d3a0cb2cd07220/frontends/election-manager/src/components/smartcard_modal/pins.test.ts#L36

Reply:
Really good suggestion! Just pushed up those changes, lmk if you have further feedback.